avatars/models.py

```diff
@@ -1,10 +1,11 @@
-# API Version 0.3.7
```

Comment:
Can you add those files to the generated list?

Reply:
I like to see the changes, especially when we have merge conflicts.